client/hosting-overview/components/plan-card.tsx

```diff
@@ -256,6 +258,23 @@ const PlanCard: FC = () => {
 						</PlanStorage>
 					</>
 				) }
+				{ isAgencyPurchase && (
```

Comment:
Should we show this only if a site is managed by A4A? i.e do:

```suggestion
				{ isA4A && (
```

I'm not sure what exactly is `isAgencyPurchase`

Reply:
The `isAgencyPurchase` checks if the plan was bought through agency.
The  `isA4A` check is meant to check if the user is an agency manager or an agency client. 
For agency clients, who don't have access to the agency dashboard, we will be still listing the site but we don't want to link to the agency dashboard that they don't have access to.
In this case, I think the logic is ok here.

Reply:
Got it. Makes sense. Thanks for the context!